src/application/services/useNoteSettings.ts

```diff
@@ -69,6 +96,10 @@ export default function (): UseNoteSettingsComposableState {
    */
   const load = async (id: NoteId): Promise<void> => {
     noteSettings.value = await noteSettingsService.getNoteSettingsById(id);
+    const response = await noteService.getNoteById(id);
```

Comment:
maybe noteWithParent?

Reply:
Maybe left response? Because parent note is not always presented. Also, there are other data like access rights